CHANGELOG.md

```diff
@@ -1,5 +1,13 @@
 # Changelog
 
+## v1.17.1 (2021-11-12)
+
+### Bug Fixes
+
+- TestCafe incorrectly reads the 'reporter' configuration file option ([#6665](https://github.com/DevExpress/testcafe/issues/6665), [#6594](https://github.com/DevExpress/testcafe/issues/6594)).
+- An error report displays multiple warnings when you debug a test in headless mode ([#6605](https://github.com/DevExpress/testcafe/issues/6605)).
```

Comment:
Any error reporter? Are there multiple error reposrters?

Reply:
Any error report. Why reporters? Should I update to the following?

An error report contains multiple warnings when you debug a test in headless mode